<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiFind</class>
 <widget class="QWidget" name="WiFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>28</height>
   </rect>
  </property>
  <property name="accessibleName">
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wiPainted" native="true">
     <property name="styleSheet">
      <string notr="true">#wiPainted { background-color: lavenderblush; }</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,0,0,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="ElidedLabel" name="lbCaption">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinIndex">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCount">
        <property name="text">
         <string notr="true">/[13]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btBack">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btNext">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbSpace">
        <property name="text">
         <string notr="true"> </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btClose">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string notr="true">Ã—</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>ElidedLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
